### PR TITLE
Improve 'unauthenticated' error message

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Improve 'unauthenticated' error message to indicate the problem is named cookie.

@spadgett

### CODE REVIEW

File: pkg/auth/auth_openshift.go [hunk 1]
```diff
@@ -177,7 +177,7 @@ func getOpenShiftUser(r *http.Request) (*User, error) {
 		return nil, err
 	}
 	if cookie.Value == "" {
-		return nil, fmt.Errorf("unauthenticated")
+		return nil, fmt.Errorf("unauthenticated, no value for cookie %s \n", openshiftSessionCookieName)
```
Comment (spadgett) [inline pkg/auth/auth_openshift.go]: Is this adding an extra newline? Not sure we need that

**Comment by benjaminapetersen** [comment]
@spadgett sorry, habit with `\n` on the `errorf`, updated!

**Comment by enj** [comment]
/uncc

@stlaz @sttts @mfojtik

**Comment by spadgett** [comment]
/override ci/prow/e2e-gcp
test has since been removed from the repo

**Pull request merged**